### PR TITLE
[feature]: update packages

### PR DESCRIPTION
**Pull request by nxjiten**
@natabal

Update the react v16.6.3 and its dependent packages. Please have a look.

### CODE REVIEW

**Comment by natabal** [comment]
@nxjiten You can create pull requests against the upstream from your branch in your fork!

File: src/components/Visualization/index.js [hunk 1]
```diff
@@ -498,8 +498,7 @@ class VisualizationView extends React.Component {
 
 
         const queryParams = Object.assign({}, context, {fullScreen:null});
-        const queryString = $.param(queryParams);
-        const iframeText = "<iframe src=\"" + window.location.origin + "/reports/visualizations/" + configuration.id + "?" + queryString + "\" width=\"800\" height=\"600\"></iframe>";
+        const iframeText = "<iframe src=\"" + window.location.origin + "/visualizations/" + configuration.id + "?" + $.param(queryParams) + "\" width=\"800\" height=\"600\"></iframe>";
```
Comment (natabal) [inline src/components/Visualization/index.js]: Do not remove 'reports' from the path.

File: src/components/Visualization/index.js [hunk 2]
```diff
@@ -517,10 +516,11 @@ class VisualizationView extends React.Component {
 
                 <Link
                     style={style.cardTitleIcon}
-                    to={{ pathname:"/reports/visualizations/" + configuration.id, query: queryParams }}
+                    to={{ pathname:"/visualizations/" + configuration.id, search: queryString.stringify(queryParams) }}
```
Comment (natabal) [inline src/components/Visualization/index.js]: same here

**Comment by nxjiten** [comment]
> @nxjiten You can create pull requests against the upstream from your branch in your fork!

Hi Natalia,
created the PR against upstream - https://github.com/nuagenetworks/visualization-framework/pull/807

**Pull request merged**